resolver/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "commonware-resolver"
+edition = "2021"
+publish = true
+version = "0.0.2"
```

Comment:
This is going to require me to release an update first to publish (because of the type changes we made). Would you mind keeping this at 0.0.1 for merge?

Reply:
When I do that update, I'll add it to the homepage/root README.